File: stage_0_ vincent main.r
#create vectors
x=c("@vincent")
y=c("@vincentayobami")
#find Hamming_Distance Between vectors
hamming <- function(x, y, na.rm = TRUE) {
  size <- 1:max(length(x) & length(y))
  x <- x[size]
  y <- y[size]
  if (na.rm) {
    del <- is.na(x) & is.na(y)
    x <- x[del]
    y <- y[del]
  }
  sum(!((x != y) %in% FALSE))
}
Name=c("Lawal Vincent Ayobami")
Email=c("vincentlawal21@gmail.com")
Slack=c("@vincent")
Biostack=c("Genomics")
Twitter=c("@vincentayobami")
Hamming_Distance=c("undefined")
df=data.frame(Name, Email, Biostack, Twitter,Hamming_Distance)
print(df)

File: stage_0_ vincent main.py
def hamming(slack,twitter):
    result=0
    if len(slack)!=len(twitter):
        print("String are not equal")
    else:
        for x,(i,j) in enumerate(zip(slack,twitter)):
            if i!=j:
                print(f'char not math{i,j}in {x}')
                result+=1
    return result
name="Lawal Vincent Ayobami"
email="vincentlawal21@gmail.com"
slackid="@vincent"
Biostack="Genomics"
twitterid="@vincentayobami"
print('Name :',name)
print('Email :',email)
print('Slack ID:',slackid)
print('Biostack :',Biostack)
print('Twitter ID:',twitterid)
print("Hamming Distance between Slack and Twitter:",hamming(slackid,twitterid))